distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.web.cfg

```diff
@@ -38,3 +38,15 @@ org.ops4j.pax.web.ssl.keypassword=${javax.net.ssl.keyStorePassword}
 
 # Client authenticate at server is wanted
 org.ops4j.pax.web.ssl.clientauthwanted=true
+
+#Included SSL/TLS Cipher Suites Regular Expressions
```

Comment:
We can, I just copied the exact description of the fields out of their metatype, because that is all that you'll get in the Admin Console as well.